application/src/main/java/net/corda/v5/application/interop/binding/InteropAction.java

```diff
@@ -9,6 +9,9 @@
  * Servers will return a [ServerResponse], wrapping the result value directly.
  */
 public abstract class InteropAction<T> {
```

Comment:
Might be worth reviewing the definition of InterOpAction as it seems a little unclear to me.